tslearn/metrics.py

```diff
@@ -672,6 +672,13 @@ def cdist_dtw(dataset1, dataset2=None, global_constraint=None,
         ``-1`` means using all processors. See scikit-learns'
         `Glossary <https://scikit-learn.org/stable/glossary.html#term-n-jobs>`_
         for more details.
+    verbose : int, optional (default=0)
+        The verbosity level: if non zero, progress messages are printed. 
+        Above 50, the output is sent to stdout. 
+        The frequency of the messages increases with the verbosity level. 
+        If it more than 10, all iterations are reported.
+        `Glossary <https://joblib.readthedocs.io/en/latest/parallel.html#parallel-reference-documentation>`_
```

Comment:
You can add the following code at the end of the docstring to remove the check for the line too long (line 707):
```
# noqa: E501
```
This will ignore this specific error in the docstring (you cannot do much about long URLs...)